plugins/polio/js/src/domains/Campaigns/Rounds/RoundsForm.tsx

```diff
@@ -1,33 +1,24 @@
-import React, { useState, useEffect } from 'react';
-import isEqual from 'lodash/isEqual';
-import { useSafeIntl } from 'bluesquare-components';
-import { Tabs, Tab, Box, Button, Tooltip, IconButton } from '@mui/material';
 import AddIcon from '@mui/icons-material/Add';
 import RemoveIcon from '@mui/icons-material/Clear';
+import { Box, Button, IconButton, Tab, Tabs, Tooltip } from '@mui/material';
+import { useSafeIntl } from 'bluesquare-components';
 import { useFormikContext } from 'formik';
+import isEqual from 'lodash/isEqual';
+import React, { FunctionComponent, useEffect, useState } from 'react';
 
 import MESSAGES from '../../../constants/messages';
-import { RoundForm } from './RoundForm.tsx';
+import { Campaign, Round } from '../../../constants/types';
 import { useStyles } from '../../../styles/theme';
+import { ScopeForm } from '../Scope/ScopeForm';
+import { RoundForm } from './RoundForm';
 
 const maxRoundsCount = 6;
 
-export const roundFormFields = rounds => {
+export const roundFormFields = (rounds: Round[]): string[] => {
     const roundKeys = [
         ...rounds
             .map((_round, i) => {
                 return [
-                    `rounds[${i}].awareness_of_campaign_planning`,
-                    `rounds[${i}].im_percentage_children_missed_in_plus_out_household`,
-                    `rounds[${i}].im_percentage_children_missed_out_household`,
-                    `rounds[${i}].im_percentage_children_missed_in_household`,
-                    `rounds[${i}].main_awareness_problem`,
-                    `rounds[${i}].lqas_district_failing`,
-                    `rounds[${i}].lqas_district_passing`,
-                    `rounds[${i}].lqas_ended_at`,
-                    `rounds[${i}].lqas_started_at`,
-                    `rounds[${i}].im_ended_at`,
-                    `rounds[${i}].im_started_at`,
```

Comment:
It seems strange to me that this hasn't been moved to something like `evaluationFormFields`